spec/auth/auth_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+require "fedex_rest_api/auth"
+require "debug"
```

Comment:
Do we need to require `debug` to run these specs?

Reply:
No, we don't. Remnants from my testing!